public/app/percona/add-instance/components/AddRemoteInstance/FormParts/FormParts.test.tsx

```diff
@@ -123,6 +123,8 @@ xdescribe('getAdditionalOptions ::', () => {
     const fields = root.find('input');
 
     expect(root.find('input[name="qan_mongodb_profiler"]').length).toBe(1);
+    expect(root.find('input[name="disable_collectors"]').length).toBe(1);
+    expect(root.find('input[name="collections_limit"]').length).toBe(1);
```

Comment:
Can we use data-testid for this fields?
(also when you came across tests using Enzyme you can try to refactor them with testing-library)

Reply:
I think yes, i will change it